doc/source/whatsnew/v0.24.0.txt

```diff
@@ -63,7 +63,8 @@ Removal of prior version deprecations/changes
 Performance Improvements
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-- Improved performance of :func:`Series.describe` in case of numeric dtpyes (:issue:`21274`)
```

Comment:
looks like u removed another entry?

Reply:
I didn't notice that. It is a mistake I made by accident.